### ISSUE TITLE
Feature Requested: A more friendly network output interface

### ISSUE THREAD

**Issue by fufjvnvnf** [stevenygd/NN.jl#6]
Right now, prediction for a model given an input is outputted by the loss function, which is unintuitive since loss function is technically not part of the net. We should consider tensorflow's implementation to output prediction else where.
